docs/rst/tutorials/core/deployment/wifi_issues_tutorial/wifi_issues_tutorial.rst

```diff
@@ -0,0 +1,352 @@
+
+You will also need the ROS package ``image_tools``, which can be installed by running:
+
+.. code:: bash
+
+    sudo apt-get install ros-jazzy-image-tools
+
+Lastly, ensure you have a working webcam connected to both of the hosts.
+
+Considerations for Video Streaming
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Before proceeding with the tutorial, consider these factors and tips to optimize video streaming over WiFi:
+
+1. Network Bandwidth:
+    To support video streaming over WiFi, ensure your network bandwidth meets the demands of the video’s bitrate, especially if multiple devices are connected.
+    Higher-resolution or higher-frame-rate video streams demand more bandwidth, so consider compressing the video or reducing the resolution if the WiFi network has limitations.
+    Future tutorials will cover ways to adjust these settings in ROS 2.
+    Additionally, you can `adjust the socket buffer sizes <https://fast-dds.docs.eprosima.com/en/latest/fastdds/use_cases/large_data/large_data.html#tuning-socket-buffer>` in your system’s network settings to accommodate larger data rates, which can help stabilize high-bandwidth streams.
+
+2. Network Congestion:
+    Multiple devices using the same network can lead to congestion, resulting in buffering or dropped frames during video streaming.
+    Tools like ``netstat`` or ``iptraf`` can monitor current network traffic, helping you assess if limiting other network activities—such as downloads or additional video streams—would improve performance.
+
+3. Video Compression and Encoding:
+    Utilizing efficient video codecs can significantly reduce the amount of data needed for transmission without compromising quality.
+    If high-quality video isn't essential, try lowering the encoding bitrate, which can significantly reduce bandwidth requirements.
+
+Run this tutorial
+------------------
+
+Configure Discovery Server entities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On Host A, start by launching the ROS 2 Discovery Server.
+Open a terminal and configure the Fast DDS server as follows:
+
+.. code-block:: bash
+
+    docker run \
+        -it \
+        --name discovery_server_container \
+        --privileged \
+        --net host \
+        --ipc host \
+        -e DISPLAY=$DISPLAY \
+        -v /tmp/.X11-unix:/tmp/.X11-unix \
+        ubuntu-vulcanexus:jazzy-desktop
+
+Once the container is running, configure Fast DDS *server* using the `Fast DDS Discovery CLI <https://fast-dds.docs.eprosima.com/en/latest/fastddscli/cli/cli.html#discovery>`__.
+
+.. code-block:: bash
+
+    fastdds discovery -t 192.168.1.165 -q 42100 -t 127.0.0.1 -q 42101
+
+The output should look similar to the following:
+
+.. code-block:: bash
+
+    ### Server is running ###
+    Participant Type:   SERVER
+    Security:           NO
+    Server ID:          0
+    Server GUID prefix: 44.53.00.5f.45.50.52.4f.53.49.4d.41
```

Comment:
The GUID should be different than this one. Please update it as the output changes when no id is specified.